admin/src/components/CKEditorInput/index.js

```diff
@@ -33,6 +34,25 @@ const CKEditorInput = ({
   const strapiTheme = localStorage.getItem("STRAPI_THEME");
   const GlobalStyling = getGlobalStyling(strapiTheme);
 
+  const [mediaLibVisible, setMediaLibVisible] = useState(false);
+
+  const handleToggleMediaLib = () => setMediaLibVisible(prev => !prev);
+
+  const handleChangeAssets = assets => {
+    let newValue = value ? value : '';
+
+    assets.map(asset => {
+      if (asset.mime.includes('image')) {
+        const imgTag = `<p><img src="${asset.url}" alt="${asset.alt}"></img></p>`;
+
+        newValue = `${newValue}${imgTag}`
+      }
+    });
+
+    onChange({ target: { name, value: newValue } });
```

Comment:
Unfortunately this is... wrong. AFAICS this replaces the entire editor content with the old content + the images.

This is neither optimal, nor insert these images at the caret position.

The idea would be to use this command: [https://ckeditor.com/docs/ckeditor5/latest/api/module_image_image_insertimagecommand-InsertImageCommand.html](https://ckeditor.com/docs/ckeditor5/latest/api/module_image_image_insertimagecommand-InsertImageCommand.html)  
  
But I failed to connect this callback with the editor instance. Probably due to my poor React exp.

Reply:
Doh, had to use the hook obviously. It works now.